src/core/models.py

```diff
@@ -1055,36 +1055,38 @@ def onetaskers(self):
                 users.append(user)
         return users
 
-    def all_editors(self):
-        press_editors = self.press_editors.all()
-        book_editors = self.book_editors.all()
+    def get_series_editor(self):
+        """Gets the editor of this book's series, if it is part of one.
 
+        Returns:
+            User: the book's series editor.
+
```

Comment:
Good docstrings, thanks for this. We don't need an empty line before closing them :)